subo/command/se2_migrate_storage.go

```diff
@@ -0,0 +1,45 @@
+package command
+
+import (
+	"context"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/suborbital/subo/subo/util"
+)
+
+// SE2MigrateStorageCommand returns the dev command.
+func SE2MigrateStorageCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "migrate [storage root]",
+		Short: "reformat storage to match SE2 specification",
+		Long:  `Update storage format to match the Suborbital Extension Engine (SE2) storage specification`,
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			location := args[0]
+
+			switch scheme(location) {
```

Comment:
shouldn't this be the thing with the call to `ToLower`?

Reply:
yes 🤦